CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## [v0.18.1](https://github.com/operator-framework/operator-lifecycle-manager/tree/v0.18.1) (2021-05-05)
+
+[Full Changelog](https://github.com/operator-framework/operator-lifecycle-manager/compare/v0.18.1...v0.18.1)
+
+**Other changes:**
+
+- Fix mishandling of log format strings in Kind test provisioner. [\#2141](https://github.com/operator-framework/operator-lifecycle-manager/pull/2141) ([benluddy](https://github.com/benluddy))
+- v0.18.1 release [\#2138](https://github.com/operator-framework/operator-lifecycle-manager/pull/2138) ([exdx](https://github.com/exdx))
+- .github/workflows: Enable GH action that retests failed workflows [\#2135](https://github.com/operator-framework/operator-lifecycle-manager/pull/2135) ([timflannagan](https://github.com/timflannagan))
+
+## [v0.18.1](https://github.com/operator-framework/operator-lifecycle-manager/tree/v0.18.1) (2021-05-04)
```

Comment:
The changelog generator did something weird here, there's a lot of overlap with the duplicate v0.18.1 section.

Reply:
I ran 
```
github_changelog_generator -u operator-framework -p operator-lifecycle-manager --since-tag=v0.18.0 --token=<token> --future-release=v0.18.1 --pr-label="**Other changes:**" -b CHANGELOG.md
```

Reply:
It seems to sometimes split the commits across days, not entirely sure why

Reply:
It must be confused about something. The diff link in the first section above is https://github.com/operator-framework/operator-lifecycle-manager/compare/v0.18.1...v0.18.1.